src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -164,4 +164,12 @@ public void testEmptyStatement() throws Exception {
         testByConfig("EmptyStatement/patchedline/defaultContextConfig.xml");
         testByConfig("EmptyStatement/context/defaultContextConfig.xml");
     }
+
+    @Test
+    @Ignore("until https://github.com/checkstyle/patch-filters/issues/150")
```

Comment:
please explain in the issue what problem we have and why this test is ignored

Reply:
done

Reply:
still not done

Reply:
@baratali I add it in https://github.com/checkstyle/patch-filters/pull/153#issue-456170764